test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9576,6 +9617,77 @@ def create_vnodes(self, name=None, attrib=None, num=1, mom=None,
             self.logger.error("name, attributes, and mom object are required")
             return False
 
+        if mom.is_cpuset_mom():
+            nodes = self.status(NODE)
+            del nodes[0]  # remove natural node]
+            if len(nodes) < num:
+                self.skipTest('cpuset mom')
```

Comment:
Can we add more descriptive reason? like 'Given mom is cpuset mom, and alteration of vnodes count is not allowed on it`